src/test/java/seedu/address/logic/parser/FindTagsCommandParserTest.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.FindTagsCommand;
+import seedu.address.model.person.PersonTagsContainsTagsPredicate;
+import seedu.address.model.tag.Tag;
+
+public class FindTagsCommandParserTest {
```

Comment:
Have you considered adding test cases for
- parse single tag 
- parse invalid tag

Reply:
By invalid tag do you mean like a tag that doesn't exist?